tests/shared/src/test/scala/munit/DoubleAssertionsSuite.scala

```diff
@@ -0,0 +1,126 @@
+package munit
+
+class DoubleAssertionsSuite extends BaseSuite {
```

Comment:
You may want to convert this test suite into a "framework suite", which makes it easier to assert success/fail results. Example https://github.com/scalameta/munit/blob/af3931c3f18d9c79fd72f06fa7734e03c3d140b3/tests/shared/src/main/scala/munit/FailFrameworkSuite.scala

Reply:
You'll need to manually register the test here https://github.com/scalameta/munit/blob/af3931c3f18d9c79fd72f06fa7734e03c3d140b3/tests/shared/src/test/scala/munit/FrameworkSuite.scala#L29